code/modules/mod/modules/_module.dm

```diff
@@ -99,6 +99,14 @@
 	if(((!mod.active || mod.activating) && !allowed_inactive) || !mod.get_charge()) //SKYRAT ADDITION: INACTIVE USE
 		balloon_alert(mod.wearer, "unpowered!")
 		return FALSE
+	// SKYRAT EDIT START - No using modules when not all parts are deployed.
+	if(!allowed_inactive)
+		for(var/obj/item/part as anything in mod.mod_parts)
+			if(part.loc == mod)
```

Comment:
early continue

Reply:
This is already an early return, making it an early-continue would just make the proc bigger and make it less obvious what it aims to do.